lib/astarte_core/cql_utils.ex

```diff
@@ -167,4 +167,35 @@ defmodule Astarte.Core.CQLUtils do
 
     eid
   end
+
+  @spec realm_name_to_keyspace_name(nonempty_binary(), nonempty_binary()) :: nonempty_binary()
+  def realm_name_to_keyspace_name(realm_name, astarte_instance_id \\ "default")
+      when is_binary(realm_name) do
+    case :persistent_term.get(realm_name, nil) do
```

Comment:
https://www.erlang.org/doc/man/persistent_term#best-practices-for-using-persistent-terms

You should not use the bare realm as key since that might cause conflicts given that persistent terms are global. You should at least namespace it with an atom, e.g. `{:astarte_realm_to_keyspace, realm_name}`